packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/paths.dart

```diff
@@ -126,8 +125,33 @@ class RetainingPathTable extends StatelessWidget {
   final bool isDiff;
   final ClassData classData;
 
-  late final _data =
-      classData.byPath.keys.map((path) => PathData(classData, path)).toList();
+  @visibleForTesting
+  static void resetSingletons() {
+    debugDataCalculationCount = 0;
+    debugDataCalculationMcs = 0;
+  }
+
+  @visibleForTesting
+  static int debugDataCalculationCount = 0;
```

Comment:
We don't need this. Checking `debugDataCalculationMicros` is positive is sufficient.